admin-dev/themes/new-theme/js/app/cldr/number-formatter.js

```diff
@@ -199,16 +201,29 @@ class NumberFormatter {
    */
   replaceSymbols(number) {
     const symbols = this.numberSpecification.getSymbol();
-    let num = number;
-    num = num.split(DECIMAL_SEPARATOR_PLACEHOLDER).join(symbols.getDecimal());
-    num = num.split(GROUP_SEPARATOR_PLACEHOLDER).join(symbols.getGroup());
-    num = num.split(MINUS_SIGN_PLACEHOLDER).join(symbols.getMinusSign());
-    num = num.split(PERCENT_SYMBOL_PLACEHOLDER).join(symbols.getPercentSign());
-    num = num.split(PLUS_SIGN_PLACEHOLDER).join(symbols.getPlusSign());
-
-    return num;
+
+    const map = {};
+    map[DECIMAL_SEPARATOR_PLACEHOLDER] = symbols.getDecimal();
+    map[GROUP_SEPARATOR_PLACEHOLDER] = symbols.getGroup();
+    map[MINUS_SIGN_PLACEHOLDER] = symbols.getMinusSign();
+    map[PERCENT_SYMBOL_PLACEHOLDER] = symbols.getPercentSign();
+    map[PLUS_SIGN_PLACEHOLDER] = symbols.getPlusSign();
+
+    return this.strtr(number, map);
   }
 
+  /**
+   * strtr() for JavaScript
+   * Translate characters or replace substrings
+   */
+  strtr(str, pairs) {
```

Comment:
I'm note sure if jsdocs are as mandatory as phpdocs ?

If that's the case, there should be **@param string str**  etc... comment above this function.

Reply:
:+1: 
Because it's a doc block, I don't regenerate assets.